Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css

```diff
@@ -27,6 +27,11 @@ td.tabbedMainMenuCell
   background-color: var(--color-contrast);
 }
 
+.tabbedMainMenu li, .tabbedSubMenu li
```

Comment:
There's a cleaner solution in CSS available: you can use `flex-grow: unset` to clear the not-wanted value. For selector, please use the original selector and prefix with tabbedMainMenu / tabbedSubMenu. That way, the selector weight is better guaranteed.

We should also add the `justify-content: start` attribute to make things more explicit.

By using `display: flex` here, you start a new flexbox, which adds an additional nesting that is not needed and results more in a workaround. The best choice when overriding problematic rules is to find the correct way to remove them (either by unsetting them or by setting a new value that correctly expresses the intention).

Here's also good information on selectors:  https://www.w3schools.com/css/css_selectors.asp & https://www.w3schools.com/cssref/css_selectors.asp
and specificity: https://www.w3schools.com/css/css_specificity.asp

And iniitial/inhertit/unset of values: https://www.w3schools.com/cssref/css3_pr_all.asp